CONTRIBUTING.md

```diff
@@ -130,7 +133,7 @@ If you want to regularly contribute to the CockroachDB docs, there are a few thi
 
 Once you've installed Jekyll and have a local clone of the docs repository, you can build and test the docs as follows:
 
-1. From the root directory of your clone, :
+1. From one of the subdirectories of the `src` directory of your clone:
```

Comment:
I tried this from `src/archive` and it did not work.

Reply:
I'll add that as an exception.